A/Agama/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+using BinaryBuilder, Pkg
+
+name = "Agama"
+repo = "https://github.com/GalacticDynamics-Oxford/Agama.git"
+
+version = v"1.0"
+sources = [
+    GitSource(repo, "f392b9d6e59269d451b37b3ada2508a44bd3a80c"),
+]
+
+platforms = supported_platforms(;exclude=x->Sys.iswindows(x) || Sys.isfreebsd(x) || (libc(x) == "musl"))
+platforms = expand_cxxstring_abis(platforms)
+
+products = [
+    LibraryProduct("libagama", :libagama),
+]
+
+dependencies = [
+    Dependency("GSL_jll"),
```

Comment:
GSL always breaks the ABI when they change the minor version, at least when they remember to do that, so specifying the compat bound is very important
```suggestion
    Dependency("GSL_jll"; compat="~2.8.1"),
```